.github/workflows/deployment.yml

```diff
@@ -26,3 +26,28 @@ jobs:
         with:
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+      - name: Login to Docker Hub
+        id: login-dockerhub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+        
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+
+      - name: Build, tag, and push image to Docker Hub
+        id: build-image
+        context: .
+        env:
+          DOCKERHUB_NAMESPACE: ${{ steps.login-dockerhub.outputs.registry }}
+          IMAGE_TAG: ${{ steps.meta.outputs.tags }}
+        run: |
+          # Build a docker container and push it to Docker Hub
+          docker build -t $DOCKERHUB_NAMESPACE/timezone-converter:$IMAGE_TAG .
+          docker push $DOCKERHUB_NAMESPACE/timezone-converter:$IMAGE_TAG
+          echo "::set-output name=image::$DOCKERHUB_NAMESPACE/timezone-converter:$IMAGE_TAG"
```

Comment:
1. Is the docker package included in the Github runners? I am looking at the `docker/build-push-action` and seems like some kind of setup is required (or at least they are doing it another way). Source: https://github.com/marketplace/actions/build-and-push-docker-images
1. What's the format of `${{ steps.meta.outputs.tags }}`? As per the syntax seems like it would be an array or something. Can we make sure we can use it like that?.

Reply:
1. I have modified this section @ibLeDy  
2. Yes we can use it like that.

Please let me know if there are any other changes needed. Thank you